lib/listen/adapter/bsd.rb

```diff
@@ -7,7 +7,7 @@
 module Listen
   module Adapter
     class BSD < Base
-      OS_REGEXP = /bsd|dragonfly/i
+      OS_REGEXP = /bsd|dragonfly/i.freeze
```

Comment:
This file can not use `frozen_string_literal`?

Reply:
The magic comment is there, but doesn't affect regular expressions. `rubocop` knows that regular expression literals are mutable, although I don't understand how.
https://github.com/rubocop-hq/rubocop/issues/6331

Reply:
I looked at the issue you mentioned, and I think `rubocop` differentiates while frozing literal type, so the change is required. But I will review if it actually is what i'm thinking.